### PR TITLE
chore(compute): include Go GAPIC targets

### PR DESCRIPTION
**Pull request by noahdietz**

### CODE REVIEW

File: google/cloud/compute/v1/BUILD.bazel [hunk 1]
```diff
@@ -217,6 +217,55 @@ java_gapic_assembly_gradle_pkg(
...
+    ],
+)
+
+go_gapic_library(
+    name = "compute_go_gapic",
+    srcs = [":compute_proto_with_info"],
+    grpc_service_config = "compute_grpc_service_config.json",
+    importpath = "cloud.google.com/go/compute/apiv1;compute",
+    service_yaml = "compute_v1.yaml",
+    transport = ["rest"],
```
Comment (vam-google) [inline google/cloud/compute/v1/BUILD.bazel:246]: Is it already supported by the rules? If yes, then great, I just missed the moment when it was added.

File: google/cloud/compute/v1/BUILD.bazel [hunk 2]
```diff
@@ -217,6 +217,55 @@ java_gapic_assembly_gradle_pkg(
...
+
+go_test(
+    name = "compute_go_gapic_test",
+    srcs = [":compute_go_gapic_srcjar_test"],
+    embed = [":compute_go_gapic"],
+    importpath = "cloud.google.com/go/compute/apiv1",
+)
+
+# Open Source Packages
+go_gapic_assembly_pkg(
```
Comment (vam-google) [inline google/cloud/compute/v1/BUILD.bazel:260]: Just as a sanity check - we do not need a transport parameter for assembly, do we (mayb e neded to generated proper build files and stuff)?

**Review by vam-google — APPROVED** [review]
LGTM, just a few questions.

File: google/cloud/compute/v1/BUILD.bazel (hunk 1 above)
Comment (noahdietz) [reply google/cloud/compute/v1/BUILD.bazel:246]: Yep, it has been supported for some time, it is just omitted from build_gen.

File: google/cloud/compute/v1/BUILD.bazel (hunk 2 above)
Comment (noahdietz) [reply google/cloud/compute/v1/BUILD.bazel:260]: Nope we do not need transport parameter for assembly. The `transport` param just toggles which transport interface implementations to generate. REST and/or gRPC. The code all goes into the same files, etc.

**Comment by jirayupiwon8** [comment]
example.v1.Messaging.GetMessage

**Review by Dortega2250 — COMMENTED** [review]
Ayuda

**Review by a2425rdl — APPROVED** [review]

**Pull request closed without merging**